src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -2,27 +2,157 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <algorithm>
 #include <chrono>
-#include <cmath>
 #include <string>
 #include <vector>
 
 #include "ngraph_transformations/op/interaction.hpp"
 #include "interaction.h"
-#include "utils/general_utils.h"
 #include <onednn/dnnl.h>
 #include <dnnl_extension_utils.h>
 #include "nodes/common/cpu_convert.h"
 #include "memory_desc/cpu_memory_desc_utils.h"
 #include "memory_desc/dnnl_blocked_memory_desc.h"
 #include "common/bfloat16.hpp"
 #include "common/cpu_memcpy.h"
+#include <ie_ngraph_utils.hpp>
 #include <cpu/x64/cpu_isa_traits.hpp>
+#include <cpu/x64/jit_generator.hpp>
+#include "emitters/jit_dnnl_emitters.hpp"
+#include "emitters/jit_load_store_emitters.hpp"
 
 namespace ov {
 namespace intel_cpu {
 namespace node {
+using namespace Xbyak;
+#define THROW_ERROR IE_THROW() << getTypeStr() << " node with name '" << getName() << "' "
+template <cpu_isa_t isa>
+struct jit_move_scale_kernel : public jit_uni_move_scale_kernel, public jit_generator {
+    DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_move_scale_kernel)
+
+    explicit jit_move_scale_kernel(const jit_move_scale_compile_params& jcp) : jit_uni_move_scale_kernel(jcp), jit_generator(jit_name()) {
+        vec_size = dnnl::impl::cpu::x64::cpu_isa_traits<isa>::vlen / sizeof(float);
+    }
+    virtual ~jit_move_scale_kernel() {}
+
+    void create_ker() override {
+        jit_generator::create_kernel();
+        ker_ = (decltype(ker_))jit_ker();
+    }
+
+private:
+    using Vmm = typename dnnl::impl::utils::conditional3<isa == cpu_isa_t::sse41, Xmm, isa == cpu_isa_t::avx2, Ymm, Zmm>::type;
+
+    void generate() override {
+        this->preamble();
+
+#define GET_OFF(field) offsetof(jit_move_scale_call_args, field)
+        mov(reg_in, ptr[reg_params + GET_OFF(p_in)]);
+        mov(reg_out, ptr[reg_params + GET_OFF(p_out)]);
+        mov(reg_work_amount, jcp_.work_amount);
```

Comment:
In general case, it's better let work_amount as a call argument. This give the possibility to an work_amout agnostic kernel.

Reply:
Sure, will modify

Reply:
1. if this ops need support dynamic shape, shape agnostic kernel could be implemented in kernel side by proper loops. The tails processed as several scalar in tails loop. This should have very trivial perf diff as tails as a vector manner for your case, but w/o recompilation overhead. Youcan ref https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/interpolate.cpp#L335-L375
2. If your case is static, just add necessary shape in compilation params and use it in compilation stage, this is more clear.

Reply:
> 1. if this ops need support dynamic shape, shape agnostic kernel could be implemented in kernel side by proper loops. The tails processed as several scalar in tails loop. This should have very trivial perf diff as tails as a vector manner for your case, but w/o recompilation overhead. Youcan ref https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/interpolate.cpp#L335-L375
> 2. If your case is static, just add necessary shape in compilation params and use it in compilation stage, this is more clear.

Thanks a lot, current only static shape is support for this op. I will refine the code to clarify this